includes/admin/services/class-rop-pinterest-service.php

```diff
@@ -443,4 +452,39 @@ public function share( $post_details, $args = array() ) {
 		return true;
 	}
 
+	function this_image_realpath_to_uploads( $image_id = 0, $requested_size = 'large' ) {
```

Comment:
@mihaiiro for here, if we're trying to get the image path, then wouldn't `get_path_by_url` provide us with the same type of path we're trying to get? This new method might be redundant if we already have a method that can get us the image path, see here: https://github.com/Codeinwp/tweet-old-post/blob/master/includes/admin/abstract/class-rop-services-abstract.php#L560

We use that method in the twitter and linkedin service files already

Reply:
- Removed my custom methods
- Tested `get_path_by_url` and the result was positive.
- added since tag where it was required.